packages/marketplace/src/resources/snippets.ts

```diff
@@ -77,4 +77,10 @@ export default [
     "code": ".main-rootlist-rootlistItemOverlay {\n  display: none;\n}",
     "preview": "resources/assets/snippets/fix-playlist-hover.png",
   },
+  {
+    "title": "Disable Homepage Recommendation",
+    "description": "Disable Homepage Recommendation charts",
+    "code": "section[data-testid='home-page'] .main-shelf-shelf:not([aria-label='Recently played'], [aria-label='Your playlists']) { display: none !important; }",
```

Comment:
`aria-label` is not the most ideal selector as the label changes for different languages, can you look into a more inclusive one?

Reply:
we select this only by `aria-label` because the order changes frequently. The only way to select these without `aria-label` is using JS

Reply:
It's better that way but not necessarily. Take a look onto the next suggestions and we're good to go.